lib/sles4sap/ipaddr2.pm

```diff
@@ -48,6 +49,17 @@ our @EXPORT = qw(
   ipaddr2_test_other_vm
   ipaddr2_cloudinit_create
   ipaddr2_cloudinit_logs
+  ipaddr2_clean_network_peering
+  ipaddr2_azure_resource_group
+  ipaddr2_ssh_internal
+  ipaddr2_bastion_ssh_addr
+  ipaddr2_get_internal_vm_private_ip
+  ipaddr2_network_peering
+  ipaddr2_patch_system
+  get_private_ip_range
```

Comment:
Why does  it needs to be exported (public interface)? Is mind to be public or private? Can be just private?